project/Dependencies.scala

```diff
@@ -43,51 +43,51 @@ object Dependencies {
   private val googleOauth2V = "0.21.0"
   private val googleOauthClientV = "1.31.0"
   private val googleCloudResourceManagerV = "0.87.0-alpha"
-  private val grpcV = "1.30.1"
+  private val grpcV = "1.30.2"
   private val guavaV = "27.0.1-jre"
   private val heterodonV = "1.0.0-beta3"
-  private val hsqldbV = "2.4.1"
+  private val hsqldbV = "2.5.1"
   private val http4sVersion = "0.20.0-M5"
   private val jacksonV = "2.10.5"
-  private val jacksonJqV = "1.0.0-preview.20190925"
-  private val janinoV = "3.0.12"
+  private val jacksonJqV = "1.0.0-preview.20191208"
+  private val janinoV = "3.0.16"
   private val javaxActivationV = "1.2.0"
   private val jaxbV = "2.3.2"
```

Comment:
This is not upgrading jaxbV to 2.3.3 per #5632 or #5663 nor providing a Scala Steward ignore directive.

Reply:
Correct. I was hoping we could circle back to this independently after the majority of the updates were merged